pkg/runtime/reconciler.go

```diff
@@ -67,6 +68,15 @@ func (r *reconciler) BindControllerManager(mgr ctrlrt.Manager) error {
 	).Complete(r)
 }
 
+// SecretValueFromReference fetches the value of a Secret given a
+// SecretReference
+func (r *reconciler) SecretValueFromReference(
+	ref *corev1.SecretReference,
+) (string, error) {
+	// TODO(alina-kim): Implement this method :)
+	return "", nil
```

Comment:
nit: maybe we should return a `"", fmt.Errorf("not implemented")` for now ?

Reply:
even better. ++

Reply:
done.